python/cudaq/kernel/ast_bridge.py

```diff
@@ -787,28 +810,6 @@ def mlirTypeFromAnnotation(self, annotation):
         if msg is not None:
             self.emitFatalError(msg, annotation)
 
-    def argumentsValidForFunction(self, values, functionTy):
```

Comment:
argumentsValueForFunction and checkControlAndTargetTypes are replaced by local functions in visit_Call